flink-cyber/cyber-parcel/src/main/meta/permissions.json

```diff
@@ -1,14 +1,4 @@
 {
-  "bin": {
-    "user": "cybersec",
-    "group": "cybersec",
-    "permissions": "0755"
-  },
-  "bin/cs-add-enrich": {
```

Comment:
Do we need to add this back because we added cs-add-enrich back in?

Reply:
yes